### PR TITLE
Move mdstat charts near to Disks

### LINKED ISSUE #11118: MDstat near disk
As we were talking [here](https://github.com/netdata/netdata/issues/11097) we need to reorganize the charts moving everything related to disks near `Disks` submenu.

The first step of this movement will bring MDstat charts nearest Disk

### PR DESCRIPTION
**Pull request by thiagoftsm**
##### Summary
Fixes #11118

##### Component Name
MD stat
##### Test Plan

1 - Compile this Branch on a host with `proc.plugin` enabled and MD devices.
2 - Take a look on your charts, you will have your submenu like next image:

Screenshot_20210507_213214

### CODE REVIEW

**Review by vlvkobal — APPROVED** [review]

**Comment by ilyam8** [comment]
In addition to https://github.com/netdata/netdata/issues/11097#issuecomment-838223071 I think we need to add an icon for the Mdstat collector. Having a generic icon doesn't look good at all. It was ~acceptable when mdstat was somewhere at the bottom of the menu, but not now.

Screenshot_20210507_213214

**Comment by manos-saratsis** [comment]
> > Could we use the same that we use for disks?
> 
> Ask @manos-saratsis
> 
> > I will make another PR for this.
> 
> I suggest adding the icon in this PR.

@ilyam8 for now lets use the same, we need to rethink the right bar anyway

**Review by vlvkobal — DISMISSED** [review]

File: web/gui/dashboard_info.js [hunk 1]
```diff
@@ -127,6 +127,10 @@ netdataDashboard.menu = {
         info: 'Charts with performance information for all the system disks. Special care has been given to present disk performance metrics in a way compatible with <code>iostat -x</code>. netdata by default prevents rendering performance charts for individual partitions and unmounted virtual disks. Disabled charts can still be enabled by configuring the relative settings in the netdata configuration file.'
     },
 
+    'mdstat': {
+        icon: '<i class="fas fa-hdd"></i>',
```
Comment (ilyam8) [inline web/gui/dashboard_info.js:131]: Do we want it to keep appearing as `mdstat`? `mdstat` is a file, usually we don't name section like that 🤷‍♂️ 

Could be:
 - `MD devices`
 - `Software RAID`

**Review by ilyam8 — APPROVED** [review]

File: web/gui/dashboard_info.js (hunk 1 above)
Comment (thiagoftsm) [reply web/gui/dashboard_info.js:131]: I will add this comment for the main issue, and when I touch the families in the next PRs, I will rename it.

Comment (ilyam8) [reply web/gui/dashboard_info.js:131]: What families? That is just a question. What do you think @vlvkobal?

Comment (thiagoftsm) [reply web/gui/dashboard_info.js:131]: We will add the family `Filesystems` to our submenu, this was the motive I said that when I work with Families, I could bring this. 
I agree 100% that `mdstat` is not the best name.

Comment (vlvkobal) [reply web/gui/dashboard_info.js:131]: I would use `MD arrays` or `Linux RAID`. `devices` and `Software RAID` sound too general.

Comment (ilyam8) [reply web/gui/dashboard_info.js:131]: That is what it is. I got it from `man mdadm`.

> mdadm - manage MD devices aka Linux Software RAID.

Comment (ilyam8) [reply web/gui/dashboard_info.js:131]: Anyway, we are in agreement that we need something better then `mdstat` 😄 👍

Comment (thiagoftsm) [reply web/gui/dashboard_info.js]: All right, now we have `MD arrays`, please review it again and thank you for your contribution :smile: !

**Review by ilyam8 — APPROVED** [review]

**Review by vlvkobal — APPROVED** [review]

**Pull request merged**